src/bluesearch/database/article.py

```diff
@@ -630,14 +680,23 @@ class TEIXMLParser(ArticleParser):
 
     Parameters
     ----------
-    xml_content
-        The content of a TEI XML file.
+    path
+        The path to a TEI XML file.
+    is_arxiv
```

Comment:
I guess another approach would be to just try to extract the `arxiv_id` each time. And for non arxiv papers we would just catch the exception raised inside of `get_arxiv_id`.

This way there would be no change in the CLI and the user would not have to worry about it.

However, this approach is also fine. Your call.

Reply:
How likely is it to have "false matches" (files that have the arxiv naming scheme, however, are not arxiv papers)?

Reply:
Personally I would find it a bit against the "principle of least astonishment" if a TEI parser assumed by default that the file is coming from a very specific source and therefore its file path contains particular information that mat be used.

This is why I set `is_arxiv` to `False` by default.

Reply:
Sure:)